src/components/GameItem.js

```diff
@@ -4,26 +4,51 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
-    height: PropTypes.number.isRequired,
-    layer: PropTypes.number.isRequired,
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLitter: this.props.itemType === 'litter',
+      cssSpottedClass: null,
```

Comment:
Hm... Is there a way we can refactor the code so that we don't need to keep/modify `cssSpottedClass` at all or in state, but instead use `isLitter` and `spotted` together to calculate the CSS class in render?